src/conf.c

```diff
@@ -61,39 +61,46 @@ int conf_loadfile( conf_t *conf, char *file )
 {
 	int line = 0;
 	FILE *fp;
-	char s[MAX_STRING], key[MAX_STRING], value[MAX_STRING];
+	char s[MAX_STRING], key[MAX_STRING];
 
 	fp = fopen( file, "r" );
 	if( fp == NULL )
 		return( 1 );			/* Not a real failure */
 
 	while( !feof( fp ) )
 	{
-		int st;
+		char *tmp, *value = NULL;
+		int st, i;
 
 		line ++;
 
 		*s = 0;
 		fscanf( fp, "%100[^\n#]s", s );
 		fscanf( fp, "%*[^\n]s" );
 		fgetc( fp );			/* Skip newline */
-		if( strchr( s, '=' ) == NULL )
+		tmp = strchr( s, '=' );
+		if( tmp == NULL )
 			continue;		/* Probably empty? */
+
 		sscanf( s, "%[^= \t]s", key );
+
+		/* start from 1 to skip '=' char */
+		for( i = 1; tmp[i]; i ++ )
```

Comment:
Can we get rid of `i`? i.e. by doing `tmp++`...

Reply:
sure, we can. I thought using an index could be cleaner. But it seems you prefer compact code :)

Reply:
I do; plus the compiler doesn't always optimize away the index.